openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -289,6 +289,7 @@ def create_import(self, request: Request, **_kwargs) -> Response:
                 taxonomy.delete()
                 return Response(import_error, status=status.HTTP_400_BAD_REQUEST)
         except ValueError as e:
+            taxonomy.delete()
```

Comment:
@rpenido I think this issue was addressed by the recently-merged https://github.com/openedx/openedx-learning/pull/122 -- it changed the [import code to catch generic Exceptions](https://github.com/openedx/openedx-learning/pull/122/files#diff-5ba76c1e98a2c00334984b6209fa173366c6996a16478f352d21a8f62b9c8189R118), so we should never see the `ValueError` here.

We didn't tag a new release with that fix though, so will need to wait until a new release is tagged to run it with the platform.